README.md

```diff
@@ -5,3 +5,415 @@ An implementation of [Soutei](http://okmij.org/ftp/papers/Soutei.pdf).
 Not yet fit for human consumption.
 
 > It's easier to ask forgiveness than it is to get permission. --Rear Admiral Grace Murray Hopper
+
+## Overview
+
+Avaleryar is an implementation of Pimlott and Kiselyov's Soutei trust-management system, comprising
```

Comment:
Can we add a link to the paper or website here?

Reply:
Makes sense---should I delete the link on line 3, or keep both?

Reply:
Oh, sorry, somehow I missed that link.  Both works for me.